strawberryfields/backends/base.py

```diff
@@ -703,41 +707,47 @@ def prepare_gaussian_state(self, r, V, modes):
         Args:
             r (array): vector of means in xp ordering
             V (array): covariance matrix in xp ordering
-            modes (int or Sequence[int]): which mode to prepare the state in
-                If the modes are not sorted, this is take into account when preparing the state.
-                i.e., when a two mode state is prepared in modes=[3,1], then the first
-                mode of state goes into mode 3 and the second mode goes into mode 1 of the simulator.
+            modes (int or Sequence[int]): Which modes to prepare the state in.
+                If the modes are not sorted, this is taken into account when preparing the state.
+                I.e., when a two mode state is prepared in modes=[3,1], the first
+                mode of the given state goes into mode 3 and the second mode goes into mode 1.
         """
         raise NotImplementedError
 
     def get_cutoff_dim(self):
-        # pylint: disable=unused-argument,missing-docstring
         raise NotApplicableError
 
     def prepare_fock_state(self, n, mode):
-        # pylint: disable=unused-argument,missing-docstring
         raise NotApplicableError
 
     def prepare_ket_state(self, state, mode):
-        # pylint: disable=unused-argument,missing-docstring
         raise NotApplicableError
 
     def prepare_dm_state(self, state, mode):
-        # pylint: disable=unused-argument,missing-docstring
         raise NotApplicableError
 
     def cubic_phase(self, gamma, mode):
-        # pylint: disable=unused-argument,missing-docstring
         raise NotApplicableError
 
     def kerr_interaction(self, kappa, mode):
-        # pylint: disable=unused-argument,missing-docstring
         raise NotApplicableError
 
     def cross_kerr_interaction(self, kappa, mode1, mode2):
-        # pylint: disable=unused-argument,missing-docstring
         raise NotApplicableError
 
     def measure_fock(self, modes, select=None):
-        # pylint: disable=unused-argument,missing-docstring
         raise NotApplicableError
+
+    def state(self, modes=None, **kwargs):
+        r"""Returns the state of the quantum simulation, restricted to the subsystems defined by `modes`.
+
+        Args:
+            modes (int or Sequence[int] or None): Specifies the modes to restrict the return state to.
+                None returns the state containing all the modes.
+                If modes is not ordered, the returned state contains the requested modes in the given order, i.e.,
```

Comment:
Note: should this instead say 'If modes *is* ordered`? If so, needs to be updated in all other docstrings this sentence appears in.

```suggestion
                If ``modes`` is not ordered, the returned state contains the requested modes in the given order, i.e.,
```

Reply:
Actually it should say that the returned state contains the modes in the same order they are listed in `modes`. I'll change it.